### DISCUSSION TITLE
Multiple protein group assignments in DiaNN output?

### DISCUSSION THREAD

**Discussion by kabalak** [vdemichev/DiaNN#858] [General]
Hej,

Normally I am using DiaNN within FragPipe v20 and for my constellation I have a full-length protein and a generated recombinant fragment of this protein in the same samples. I supply a combined FASTA file (organism + fragment-specific FASTA) for the FragPipe SpecLib search.  The peptide.tsv file from FragPipe features the "Mapped proteins" with the fragment protein ID for the ambigous peptides.

This allows me to align these entries - on the protein-level or DiaNN output files any reference to multiple protein ID matches are not present. Can we somehow switch that on, so I can conclude the protein quantity for the protein fragment from the DiaNN output?

**Comment by vdemichev** [comment]
Sorry, can you please elaborate in more detail on what's going on and maybe attach some screenshots which illustrate things, i.e. what DIA-NN currently outputs and what you'd like it to output? Protein.Ids column lists all protein IDs, Protein.Group - inferred proteins, in most cases this is all the information needed for downstream inference.

**Reply by kabalak** [reply]
If you have multiple protein sequences matching the same peptides, you can in the peptide.tsv file see them assorted like 
![image](https://github.com/vdemichev/DiaNN/assets/75732067/b13ea7ab-d014-4baf-8eac-7c8cb1e6a393)
 under the "Mapped proteins" header. But for the DiaNN output, I do never get any multiple protein assignments for the peptides: 
![image](https://github.com/vdemichev/DiaNN/assets/75732067/3a06b35b-c303-48f6-9f10-e058bcef8c40)

Maybe this is a setting thing in combination with FragPipe, but I do never get differential Protein.Ids and Protein.Group at all.

**Reply by vdemichev** [reply]
Yes, FragPipe apparently does not use that column as input for DIA-NN. Can suggest to just match information in this report to DIA-NN report using R or Python.
